vita3k/gui/src/initial_setup.cpp

```diff
@@ -210,6 +213,14 @@ void draw_initial_setup(GuiState &gui, EmuEnvState &emuenv) {
     case SELECT_INTERFACE_SETTINGS:
         title_str = lang["select_interface_settings"];
         ImGui::SetCursorPosY((WINDOW_SIZE.y / 2.f) - ImGui::GetFontSize());
+        ImGui::Checkbox("Display Info Message", &emuenv.cfg.display_info_message);
```

Comment:
```suggestion
        ImGui::Checkbox("Display Compatibility Update Notification", &emuenv.cfg.display_info_message);
```

Reply:
Setting added is just same of setting dialog, if i change here i need change on setting dialog too 
Also is really info message, i have code it in base for can set every info message inside, not only compat

Reply:
Well for the time being it only gives notifications about the compatibility database update, feel free to update it once you make it display something else.
Also "info message" is way too broad, it is one of the first thing displayed and some user can let it checked thinking it shows important information.

Reply:
Hmm important no, juste said is info message, no santé said is important
And your suggestion is too long, info message is short